component_down.py

```diff
@@ -0,0 +1,33 @@
+import os
+import time
+import json
+from collections import defaultdict
+
+agent= os.getenv('HOSTNAME')
```

Comment:
I think we should check whether the hostname belongs to `fnal.gov` domain or `cern.ch` and then we set the jsonDir according to it. Something like
```
if 'fnal.gov' in agent:
  jsonDir = '/afs/cern.ch/user/c/cmst1/www/Automatic_Agent_json/'
else:
  jsonDir = '/data/srv/wmagent/current'
```

such that it doesn't fail if running at FNAL nodes.

Reply:
Again. I think this part works fine it was only the xrdcp command that failed at cern but if you would like extra protections I can added these lines in and cross check to make sure it works.